opent2t.p.hub/com.wink.hub/js/thingTranslator.js

```diff
@@ -284,14 +322,19 @@ class Translator {
     /**
      * Internal helper method which makes the actual request to the wink service
      */
-    _makeRequest(path, method, content) {
+    _makeRequest(path, method, content, includeBearerHeader) {
+       
+        var addBearerHeader = (typeof includeBearerHeader !== 'undefined') ?  includeBearerHeader : true;
```

Comment:
it appears that this was the defacto std way of setting a default value for function parameters ie default parameters-- so I didn't want to have to change all existing calls to have it be true so if not specified this way I can default it to true


---
In reply to: [88971888](https://github.com/openT2T/translators/pull/126#discussion_r88971888) [](ancestors = 88971888)